### PR TITLE
feat(components): Update formatDate component with optional date

### PR DESCRIPTION
**Pull request by ivan-ma-jobber**
## Motivations
This change is required for the Jobber Payments team to display the date using the `FormatDate` component without a year

### Added

- 

### Changed

- `FormatDate` component now has a new optional `showYear` argument.

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/FormatDate/FormatDate.tsx [hunk 1]
```diff
@@ -8,9 +8,23 @@ interface FormatDateProps {
...
+   */
+  readonly showYear?: boolean;
 }
 
-export function FormatDate({ date: inputDate }: FormatDateProps) {
+interface formatOptions {
+  readonly month: "short";
+  readonly day: "numeric";
+  readonly year?: "numeric";
+}
```
Comment (MichaelParadis) [inline packages/components/src/FormatDate/FormatDate.tsx:22]:
```suggestion

```

File: packages/components/src/FormatDate/FormatDate.tsx [hunk 2]
```diff
@@ -21,13 +35,19 @@ export function FormatDate({ date: inputDate }: FormatDateProps) {
...
 
-  return <>{strFormatDate(dateObject)}</>;
+  return <>{strFormatDate(dateObject, showYear)}</>;
 }
 
-export function strFormatDate(date: Date) {
-  return date.toLocaleDateString(undefined, {
-    year: "numeric",
+export function strFormatDate(date: Date, showYear = true) {
+  let formatOptions: formatOptions = {
```
Comment (MichaelParadis) [inline packages/components/src/FormatDate/FormatDate.tsx:42]:
```suggestion
  let formatOptions: Intl.DateTimeFormatOptions = {
```

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Minor type cleanup but overall looks good to me

File: packages/components/src/FormatDate/FormatDate.tsx (hunk 2 above)
Comment (ivan-ma-jobber) [reply packages/components/src/FormatDate/FormatDate.tsx]: Good suggestion made the change [here](https://github.com/GetJobber/atlantis/pull/1301/commits/c382f2be8df523efafb2e64af73f710b9cc27291)

File: packages/components/src/FormatDate/FormatDate.tsx (hunk 1 above)
Comment (ivan-ma-jobber) [reply packages/components/src/FormatDate/FormatDate.tsx]: Good suggestion made the change [here](https://github.com/GetJobber/atlantis/pull/1301/commits/c382f2be8df523efafb2e64af73f710b9cc27291)

**Pull request merged**